spec/granite/migrator/migrator_spec.cr

```diff
@@ -0,0 +1,135 @@
+require "../../spec_helper"
+
+describe Granite::Migrator::Base do
+  describe "#drop_sql" do
+    it "generates correct SQL with #{{{ env("CURRENT_ADAPTER") }}} adapter" do
+      {% if env("CURRENT_ADAPTER") == "mysql" %}
+        Review.migrator.drop_sql.should eq "DROP TABLE IF EXISTS `reviews`;"
+      {% else %}
+        Review.migrator.drop_sql.should eq "DROP TABLE IF EXISTS \"reviews\";"
+      {% end %}
+    end
+  end
+
+  describe "#create_sql" do
+    it "generates correct SQL with #{{{ env("CURRENT_ADAPTER") }}} adapter" do
+      {% if env("CURRENT_ADAPTER") == "pg" %}
+        Review.migrator.create_sql.should eq <<-SQL
+          CREATE TABLE "reviews"(
+          "id" BIGSERIAL PRIMARY KEY,
+          "name" VARCHAR(255)
+          ,
+          "downvotes" INT
+          ,
+          "upvotes" BIGINT
+          ,
+          "sentiment" FLOAT
+          ,
+          "interest" REAL
```

Comment:
Im thinking the type of `interest` should be `DOUBLE PRECISION`.  Based on the [docs](https://www.postgresql.org/docs/9.5/datatype.html#DATATYPE-TABLE) that would be their 8 byte floating point type.